src/privateStore.js

```diff
@@ -38,14 +43,16 @@ class PrivateStore {
    */
   async set (key, value) {
     if (!this.db) throw new Error('_sync must be called before interacting with the store')
+    if (key === SALT_KEY) throw new Error('Invalid key')
 
-    const encryptedValue = this._encryptEntry(value)
+    if (value != null) {
+      value = this._encryptEntry(value)
+    }
     const dbKey = this._genDbKey(key)
 
     // TODO - error handling
-    const multihash = await this.db.put(dbKey, encryptedValue)
-
-    this.updateRoot(multihash)
+    const hash = await this.db.put(dbKey, value)
```

Comment:
So are we setting the value "null" here if the user plugs in "null"? Is that intended?

Reply:
Yes, setting to `null`is the only way to remove an entry.

Reply:
Are you saying that we should encrypt `null` so that you can't see that you've removed an entry?

Reply:
@oed Is this an orbit-db convention? i.e. setting a value to null is how you delete key-value entries? If that's the case then we should probably support it as you've done.

Reply:
@christianlundkvist No I'm not sure there is a convention. Actually let me ask